go/common/log/logutil/README.md

```diff
@@ -0,0 +1 @@
+This package contains utils related to logging.
```

Comment:
Do these utils really need another package, vs just putting them in the `log` package?